Description: do not build unused clang tools
Author: Olivier Tilloy <olivier.tilloy@canonical.com>

--- a/tools/clang/scripts/build.py
+++ b/tools/clang/scripts/build.py
@@ -523,7 +523,7 @@ def main():
 
   targets = 'AArch64;ARM;Mips;PowerPC;SystemZ;WebAssembly;X86'
 
-  projects = 'clang;compiler-rt;lld;chrometools;clang-tools-extra'
+  projects = 'clang;compiler-rt;lld;chrometools'
 
   if sys.platform == 'darwin':
     # clang needs libc++, else -stdlib=libc++ won't find includes
