Description: do not build unused clang tools
Author: Olivier Tilloy <olivier.tilloy@canonical.com>
Forwarded: not-needed

--- a/tools/clang/scripts/build.py
+++ b/tools/clang/scripts/build.py
@@ -644,7 +644,7 @@ def main():
   ldflags = []
 
   targets = 'AArch64;ARM;LoongArch;Mips;PowerPC;RISCV;SystemZ;WebAssembly;X86'
-  projects = 'clang;lld;clang-tools-extra'
+  projects = 'clang;lld'
   if args.bolt:
     projects += ';bolt'
 
