Description: do not build unused clang tools
Author: Olivier Tilloy <olivier.tilloy@canonical.com>
Forwarded: not-needed

--- a/tools/clang/scripts/build.py
+++ b/tools/clang/scripts/build.py
@@ -643,7 +643,7 @@ def main():
 
   targets = 'AArch64;ARM;Mips;PowerPC;RISCV;SystemZ;WebAssembly;X86'
 
-  projects = 'clang;lld;clang-tools-extra'
+  projects = 'clang;lld'
   runtimes = 'compiler-rt'
 
   base_cmake_args = [
