Description: do not build unused clang tools
Author: Olivier Tilloy <olivier.tilloy@canonical.com>

--- a/tools/clang/scripts/build.py
+++ b/tools/clang/scripts/build.py
@@ -701,7 +701,7 @@ def main():
   # Build lld and code coverage tools. This is done separately from the rest of
   # the build because these tools require threading support.
   print('Building thread-enabled tools.')
-  tools_with_threading = [ 'dsymutil', 'lld', 'llvm-cov', 'llvm-profdata' ]
+  tools_with_threading = [ 'lld' ]
   print('Building the following tools with threading support: %s' %
         str(tools_with_threading))
 
